src/styles/blocks/calendar.css

```diff
@@ -0,0 +1,17 @@
+/* Error: Undefined variable.
+ *   ,
+ * 4 |   width: (7 * $size-day) + 2 * $padding-month;
+ *   |               ^^^^^^^^^
+ *   '
+ *   calendar.scss 4:15  root stylesheet */
+
+body::before {
+  font-family: "Source Code Pro", "SF Mono", Monaco, Inconsolata, "Fira Mono",
+      "Droid Sans Mono", monospace, monospace;
+  white-space: pre;
+  display: block;
+  padding: 1em;
+  margin-bottom: 1em;
+  border-bottom: 2px solid black;
+  content: "Error: Undefined variable.\a   \2577 \a 4 \2502    width: (7 * $size-day) + 2 * $padding-month;\a   \2502                ^^^^^^^^^\a   \2575 \a   calendar.scss 4:15  root stylesheet";
```

Comment:
The content property is displaying an error message, which is unneeded in the final CSS file. Remove this after fixing the SCSS error.